test/e2e/specs/test.spec.js

```diff
@@ -56,16 +181,16 @@ describe('Vaults UI Test Cases', () => {
             .next('.input-wrapper')
             .within(() => {
               cy.get('input[type="number"]').click();
-              cy.get('input[type="number"]').type(5);
+              cy.get('input[type="number"]').type(1);
             });
         });
 
       cy.contains('button', 'Adjust Vault').click();
       cy.confirmTransaction().then(taskCompleted => {
         expect(taskCompleted).to.be.true;
-        cy.contains('p', "Your vault's balances have been updated.").should(
-          'exist',
-        );
+        cy.contains('p', "Your vault's balances have been updated.", {
+          timeout: 60000,
```

Comment:
why are these the timeouts? are they particular to each of these tests? consider putting in a config file. E.g. this one is `MINUTE_MS`

Reply:
these timeouts are specifically placed after a transactions occurs, since on average a transaction takes more than the default 30s but less than 60s